cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -539,6 +556,8 @@ def test_validate_custom_rate_cpu_usage_cost(chargeback_costs_custom, chargeback
             break
 
 
+@pytest.mark.uncollectif(
+    lambda provider: provider.type == 'gce')
```

Comment:
Please use `provider.one_of(GCEProvider)` instead of checking type string.

Reply:
Done